Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -609,14 +609,7 @@ open class CarPlayNavigationViewController: UIViewController, BuildingHighlighti
             currentLegIndexMapped = legIndex
         }
         
-        if routeLineTracksTraversal {
-            if routeProgress.routeIsComplete {
-                navigationMapView?.removeRoutes()
-            }
-            navigationMapView?.updateUpcomingRoutePointIndex(routeProgress: routeProgress)
-            navigationMapView?.travelAlongRouteLine(to: location.coordinate)
-        }
-
+        navigationMapView?.updateRouteLine(routeProgress: routeProgress, coordinate: location.coordinate, redraw: true)
```

Comment:
Is it intended change? Would `NavigationMapView` be aware of changes to `CarPlayNavigationViewController. routeLineTracksTraversal`?

Reply:
Yes. Because when the `CarPlayNavigationViewController. routeLineTracksTraversal` change, it would set the `navigationMapView?.routeLineTracksTraversal` as the same value.